src/model/search-stream-helper-model/event-data.model.ts

```diff
@@ -0,0 +1,7 @@
+import { float } from "aws-sdk/clients/lightsail";
+
+export class EventData {
+    event_id: number | undefined;
```

Comment:
event_id?: number could solve this same issue. Is there any additional benefits of using undefined with pipe ("|") here?